src/style.scss

```diff
@@ -14,70 +14,79 @@
 // Support for auroElement styles
 // @import './node_modules/@aurodesignsystem/webcorestylesheets/dist/auroElement/auroElement';
 
-.toastButton {
-  width: var(--auro-size-400);
-  height: var(--auro-size-400);
-  padding-right: 0;
-  padding-left: 0;
-  border: 2px solid transparent;
-  margin-left: auto;
-  background-color: transparent;
-
-  &:hover {
-    border: 1px solid var(--auro-color-ui-bkg-hover-on-light);
-    border-radius: 50%;
-    background-color: var(--auro-color-ui-bkg-hover-on-light);
-    cursor: pointer;
-
-  }
-
-}
-
 // container styles - defines the look of the outer custom component
 :host([visible]) {
-  position: fixed;
-  z-index: var(--auro-depth-modal);
-  right: 2%;
-  bottom: 2%;
-  left: 2%;
   display: inline-flex;
-  min-width: 95%;
+  min-width: 100%;
   min-height: 60px;
```

Comment:
I rally want to create `var(--auro-size-750)` for this value.

Reply:
would you like me to create an issue in the Auro Design repo and create a value for that? or should i add a comment on here to update this value once that is done?

Reply:
@blackfalcon this is the response i got from designers about the min-width being 60px, "let’s keep it at 60px for now… we’ll get back to you Friday at the latest if we want to make a change if that’s okay?" designers are out at a design summit today and tomorrow. 

i'm kind of in a pickle here; i'm debating on going ahead and creating a new token for this or wait till friday to hear back from them? i would like to create one to keep things going, but i know if they end up wanting to use a pixel size that already exists as a token we won't need that token anymore. what are you thoughts?

Reply:
We're good. Not a critical step.